homework/jenya_s/homework4.py

```diff
@@ -0,0 +1,25 @@
+def find_most_frequent(text):
+
+    # making a new string in lowercase
+    # and replacing punctuation marks in it with spaces
+    sub_str = text.lower()
+    translation_table = dict.fromkeys(map(ord, ",.:;!?-"), " ")
+    sub_str2 = sub_str.translate(translation_table)
+
+    # creating a list of separate words and getting rid of spaces
+    lst = [x for x in sub_str2.split(" ") if x!=""]
```

Comment:
this can be simple `[x for x in sub_str2.split(" ") if x]`

Reply:
if x!="" part was added specifically for cases, when several punctuation marks are added in a row(just like in the example "TOm?!"). The matter is that in such cases some additional spaces are present in a list of separate words, you may test it from your side if you want.

Reply:
you ca remove `!=""` part and this will be the same